packages/contracts-bedrock/src/interop/L2CrossDomainMessenger.sol

```diff
@@ -0,0 +1,147 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { L2CrossDomainMessenger } from "src/L2/L2CrossDomainMessenger.sol";
+
+import { Constants } from "src/libraries/Constants.sol";
+import { Encoding } from "src/libraries/Encoding.sol";
+import { Predeploys } from "src/libraries/Predeploys.sol";
+import { SafeCall } from "src/libraries/SafeCall.sol";
+
+contract InteropEnabledL2CrossDomainMessenger {
```

Comment:
Nit on this name, its very verbose. `L2ToL2CrossDomainMessenger` or `SuperchainCrossDomainMessenger`?

Reply:
Since this is meant to replace the L2CrossDomainMessenger, i'll actually stick with the old name and alias the import